commerce-account-service/src/main/java/com/liferay/commerce/account/internal/util/CommerceAccountHelperImpl.java

```diff
@@ -114,18 +114,19 @@ public CommerceAccount getCurrentCommerceAccount(
 		long currentCommerceAccountId = SessionParamUtil.getLong(
 			httpServletRequest, curGroupCommerceAccountIdKey);
 
-		if (currentCommerceAccountId == 0) {
+		if (currentCommerceAccountId > 0) {
+			commerceAccount = _commerceAccountService.fetchCommerceAccount(
+				currentCommerceAccountId);
+		}
+
+		if ((currentCommerceAccountId <= 0) || !commerceAccount.isActive()) {
```

Comment:
`(currentCommerceAccountId <= 0)` should probably now be a null check, since `fetchCommerceAccount` could return null.